docs/getting-started/rulesets.md

```diff
@@ -0,0 +1,141 @@
+# Spectral Rulesets
+
+Rulesets are collections of rules, in a YAML or JSON file. These rules are taking parameters, and calling functions on certain parts of another YAML or JSON object being linted. 
```

Comment:
```suggestion
Rulesets are collections of rules and functions, in a YAML or JSON file. These rules are taking parameters, and calling functions on certain parts of another YAML or JSON object being linted. 
```

Reply:
Are they? The functions are defined in other files and only referenced from the ruleset, so I'm not sure this would be accurate?

Reply:
Yes, rules and functions all technically belong to rulesets. We don't allow people to declare functions in "ruleset files" yet, but they technically belong to the same container.

Reply:
Even when #311 is done we will not be **defining** functions in rulesets, we will only be allowing people to reference them. This is what I mean when I say im not sure this word change is correct.

Reply:
Hmm, ok gotcha, in this case it's a bit inconsistent with the JS API but we can solve that separately.